torch/_subclasses/fake_tensor.py

```diff
@@ -2774,7 +2774,7 @@ def validate(x: T) -> Union[T, FakeTensor]:
 
             nonlocal flat_arg_fake_tensors
             if not self.is_our_fake(x):
```

Comment:
Hi @Valentine233 could you please also add unit test for this change ?

Reply:
Thanks @atalman , the UT has been added.